crates/bevy_ecs/src/lib.rs

```diff
@@ -540,8 +540,8 @@ mod tests {
         let f = world
             .spawn((TableStored("def"), A(456), SparseStored(1)))
             .id();
-        // // this should be skipped
-        // SparseStored(1).spawn("abc");
```

Comment:
rebase error? `.spawn("abc")` is valid code

Reply:
Yeah looks like it. Though I'm tempted to revert the changes there since it's unrelated to anything else in this PR.

Reply:
Fixed it to use more up to date APIs.